collection-scripts/gather

```diff
@@ -43,41 +46,38 @@ resources+=(nodes)
 resources+=(machines)
 
 # Run the collection of resources using must-gather
-echo ${resources[@]} | tr ' ' '\n' | xargs -t -I{} -P 5 sh -c 'echo "inspecting $1" && oc adm inspect --dest-dir must-gather --all-namespaces $1' -- {}
+echo ${resources[@]} | tr ' ' '\n' | xargs -t -I{} -P 5 sh -c 'echo "inspecting $1" && oc adm inspect --dest-dir $2 --all-namespaces $1' -- {} ${BASE_COLLECTION_PATH}
```

Comment:
```suggestion
echo ${resources[@]} | tr ' ' '\n' | xargs -t -I{} -P 5 sh -c 'echo "inspecting $1" && oc adm inspect --dest-dir ${BASE_COLLECTION_PATH} --all-namespaces $1' -- {} ${BASE_COLLECTION_PATH}
```

Reply:
`BASE_COLLECTION_PATH` is not set within the xargs process . that why it added after the `--` and we refer it as `$2`